editor.planx.uk/src/@planx/components/PlanningConstraints/Presentational.tsx

```diff
@@ -0,0 +1,167 @@
+import ErrorOutline from "@mui/icons-material/ErrorOutline";
+import Typography from "@mui/material/Typography";
+import type { Constraint, GISResponse } from "@opensystemslab/planx-core/types";
+import Card from "@planx/components/shared/Preview/Card";
+import CardHeader from "@planx/components/shared/Preview/CardHeader";
+import capitalize from "lodash/capitalize";
+import { HandleSubmit } from "pages/Preview/Node";
+import React from "react";
+import ReactMarkdownOrHtml from "ui/shared/ReactMarkdownOrHtml";
+
+import { ErrorSummaryContainer } from "../shared/Preview/ErrorSummaryContainer";
+import SimpleExpand from "../shared/Preview/SimpleExpand";
+import { WarningContainer } from "../shared/Preview/WarningContainer";
+import ConstraintsList from "./List";
+import { DEFAULT_PLANNING_CONDITIONS_DISCLAIMER } from "./model";
+import { InaccurateConstraints } from "./Public";
+
+export type PresentationalProps = {
+  title: string;
+  description: string;
+  fn: string;
+  disclaimer: string;
+  constraints: GISResponse["constraints"];
+  metadata: GISResponse["metadata"];
+  handleSubmit: () => void;
+  refreshConstraints: () => void;
+  inaccurateConstraints: InaccurateConstraints;
+  setInaccurateConstraints: (
+    value: React.SetStateAction<InaccurateConstraints>,
+  ) => void;
+};
+
+export function Presentational(props: PresentationalProps) {
+  const {
+    title,
+    description,
+    constraints,
+    metadata,
+    disclaimer,
+    inaccurateConstraints,
+    setInaccurateConstraints,
+  } = props;
+
+  const error = constraints.error || undefined;
+  const showError = error || !Object.values(constraints)?.length;
+  if (showError) return <ConstraintsFetchError error={error} {...props} />;
+
+  const positiveConstraints = Object.values(constraints).filter(
+    (v: Constraint, _i) => v.text && v.value,
+  );
```

Comment:
question: is `_i` here used for readability?

Reply:
I can remove this - I think the unused index (aka `_i`) is an old remnant from when we used to filter positive & negative constraints _and then also_ only show the top three of each/hide the rest or something like that